content/en/docs/Endpoints/getTranscodeStream.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "getTranscodeStream"
+linkTitle: "getTranscodeStream"
+categories:
+- Transcoding
+description: >
+  Returns a transcoded media stream.
+---
+
+`http://your-server/rest/getTranscodeStream`
+
+Returns a transcoded media stream. **Note:** Clients should not construct this URL manually. Instead, they should use the `transcodeUrl` provided in the response of the [`getTranscodeDecision`](../gettranscodedecision) endpoint.
+
+### Parameters
+
+| Parameter | Req. | OpenS. | Default | Comment |
+| --- | --- | --- | --- | --- |
+| `trackID` | **Yes** | | | The ID of the track to transcode. |
+| `offset` | No | | 0 | The time offset in seconds from which to start transcoding. |
+| `parameters` | **Yes** | | | Server-specific transcoding parameters, obtained from the `getTranscodeDecision` endpoint. |
```

Comment:
`parameters` is a very broad idea. Is this something that should be _repeated`? (e.g., `parameters=...&parameters=...`). I understand that the point is probably to allow servers to be as generic as possible, but it would be worth codifying _at least_ whether it's single-valued or multi-valued.